shunting-yard-common/pom.xml

```diff
@@ -20,6 +20,13 @@
       <artifactId>commons-lang3</artifactId>
     </dependency>
 
+    <!-- Apiary -->
+    <dependency>
```

Comment:
Follow on work - let's see if we can remove this dependency from common to receiver-sqs